src/sysext/sysext.c

```diff
@@ -1673,7 +1678,7 @@ static int merge_subprocess(
         /* Order by version sort with strverscmp_improved() */
         typesafe_qsort(extensions, n_extensions, strverscmp_improvedp);
 
-        buf = strv_join(extensions, "', '");
+        buf = strv_join(extensions_v, "', '");
```

Comment:
hmm extensions[] is now sorted, but extensions_v[] isn't. We should sort it too, no?